.betterer.results

```diff
@@ -5898,6 +5898,15 @@ exports[`better eslint`] = {
     "public/app/plugins/datasource/cloudwatch/__mocks__/monarch/Monaco.ts:5381": [
       [0, 0, 0, "Unexpected any. Specify a different type.", "0"]
     ],
+    "public/app/plugins/datasource/cloudwatch/api.ts:5381": [
```

Comment:
These issues were not introduced in this PR - I've just been moving code around here. Anyway, this PR is already too big so won't be fixing the issues here. New PR including Betterer fixes will come soon. cc @joshhunt